module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp

```diff
@@ -925,6 +925,7 @@ EinkStatus_e EinkFillScreenWithColor(EinkDisplayColorFilling_e colorFill)
     buf[8] = EINK_DISPLAY_RES_Y & 0x00FF;
 
     if (BSP_EinkWriteData(buf, 9, SPI_AUTOMATIC_CS) != 0) {
+        BSP_EinkWriteCS(BSP_Eink_CS_Set);
```

Comment:
Is this change really required? AFAIR calling `EinkWriteData` with `SPI_AUTOMATIC_CS` should already leave `~CS` line in high state after the transmission has completed

Reply:
You are right. It is not needed, I will remove it